app/addons/documents/views.js

```diff
@@ -505,21 +517,37 @@ function(app, FauxtonAPI, Components, Documents, Databases, Views, QueryOptions)
     },
 
     beforeRender: function() {
-      var docs;
-
       this.removeNestedViews();
 
       this.pagination.setCollection(this.collection);
       this.allDocsNumber.setCollection(this.collection);
 
-      docs = this.expandDocs ? this.collection : this.collection.simple();
```

Comment:
i think we can delete it then, i searched in the code for it and it was the only usage i think (you might want to double check)